### DISCUSSION TITLE
怎样才能收到激活码

### DISCUSSION THREAD

**Discussion by aprjie** [getlantern/lantern-forum-cn#575] [Q&A]
注册邮箱是aprjie@msn.com 收不到激活码

**Comment by aprjie** [comment]
@Mike777Khoo 可以将激活码发送至邮箱吗

**Reply by aprjie** [reply]
![截屏 2024-06-14 22 13 29 2](https://github.com/getlantern/lantern-forum-cn/assets/172819424/ea9327d7-d762-421d-9b69-b501aa377958)

**Comment by MaxMKMax** [comment]
@aprjie 您好. 可以的. 我帮您安排!

**Reply by aprjie** [reply]
@Mike777Khoo 我到现在还没收到过邮件，或者你这边可以直接留言把激活码留下

**Reply by MaxMKMax** [reply]
@aprjie 您好. 请您提供我您的支付单据. 我帮您要激活码. 谢谢.

**Reply by aprjie** [reply]
![截屏 2024-06-14 22 13 29 2](https://github.com/getlantern/lantern-forum-cn/assets/172819424/b855ed19-e3dc-4275-b13a-8244723daee1)

**Reply by aprjie** [reply]
@Mike777Khoo 付款凭证

**Reply by MaxMKMax** [reply]
@aprjie 您的专属激活码:
CDXMQ-6QC67-TH868-MBGFD-8MGQJ

**Comment by aprjie** [comment]
@Mike777Khoo 可以给我激活码了吗

**Comment by aprjie** [comment]
@Mike777Khoo 到底什么时候能解决问题

**Reply by warm3snow** [reply]
lantern就是电信诈骗，我上个月6.19开通的lantern pro，一直收不到激活码，后来按照@Mike777Khoo给的退款方式申请退款，但是到现在也没有退款。

**Reply by MaxMKMax** [reply]
@warm3snow 我们是一家超过 10 年有信誉的公司. 不是您说的诈骗. 请您有耐心点, 退款过程需要两周才完成, 您也才一周而已. 我们的特定部门还需要几天处理的. 您在这里提供我支付单据和账号邮箱吧. 我现在就帮您去追讨.

**Comment by MaxMKMax** [comment]
@aprjie 您好. 马上给您提交问题. 给我们 3 天时间. 能尽快就尽快回您. 我帮您申请一个月补偿吧. 非常抱歉让您久等了!

**Comment by MaxMKMax** [comment]
@aprjie 帮您再次提交好了. 请您过几天查看这里或邮箱.

**Comment by MaxMKMax** [comment]
@aprjie 
您的专属激活码:
CDXMQ-6QC67-TH868-MBGFD-8MGQJ

也给您重新算新的一年了. 等于给您补偿. 谢谢您支持蓝灯!

**Reply by aprjie** [reply]
激活码不正确，能否再帮忙看一下，邮箱地址是aprjie@msn.com
![Uploading 截屏2024-07-16 15.43.37.png…]()

**Reply by MaxMKMax** [reply]
@aprjie 好的. 我帮您问一下. 我们还是用邮件发给您. 你之后查看邮件.

**Reply by aprjie** [reply]
麻烦尽快，拖了那么久了

**Reply by MaxMKMax** [reply]
@aprjie 您好. 好的没问题我会帮您提交问题给技术团队. 但要等到周三了. 我们在做维修. 不好意思让您久等了!

**Reply by aprjie** [reply]
> @aprjie您好。 好的没问题我会帮您提交问题给技术团队。 但要等到周三了。 我们在做维修。 不好意思让您久等了！

还没解决吗？

**Reply by aprjie** [reply]
> @aprjie您好。 好的没问题我会帮您提交问题给技术团队。 但要等到周三了。 我们在做维修。 不好意思让您久等了！

何时能解决啊

**Reply by aprjie** [reply]
> @aprjie 您好. 好的没问题我会帮您提交问题给技术团队. 但要等到周三了. 我们在做维修. 不好意思让您久等了!

还要等多久啊

**Comment by aprjie** [comment]
@Mike777Khoo 什么时候能帮忙解决

**Reply by MaxMKMax** [reply]
@aprjie 
您好. 一个月前给过您了啊. 在上面的回复.
CDXMQ-6QC67-TH868-MBGFD-8MGQJ

也给您重新算新的一年了. 等于给您补偿. 谢谢您支持蓝灯!

**Reply by aprjie** [reply]
> CDXMQ-6QC67-TH868-MBGFD-8MGQJ

我已经反映过这个激活码无法激活了

**Comment by aprjie** [comment]
<img width="278" alt="截屏2024-07-16 15 43 37" src="https://github.com/user-attachments/assets/a7e45d80-1f04-4325-9f50-c9466564b628">
@warm3snow

**Reply by MaxMKMax** [reply]
@aprjie 好的, 我重新帮您拿新的激活码. 会给您发邮件. 请查看.

**Reply by aprjie** [reply]
现在已经九月份了，激活码呢？ @Mike777Khoo

**Comment by MaxMKMax** [comment]
@aprjie 您好. 发过几次给您了. 不清楚为什么依然用不了. 这次我再帮您拿激活码多一次吧.

**Reply by kidokogills** [reply]
@Mike777Khoo 我也想要激活码......

**Comment by MaxMKMax** [comment]
@aprjie 激活码已经发到您的邮箱了.

**Reply by aprjie** [reply]
怎么才6个数字，不对啊
